src/app/components/team/SmoochBot/SmoochBotMainMenuOption.js

```diff
@@ -65,75 +36,72 @@ const SmoochBotMainMenuOption = ({
-              submitted && !text ?
+        <>
+          <p>
+            <TextField
+              label={
                 <FormattedMessage
-                  id="smoochBotMainMenuOption.labelError"
-                  defaultMessage="Please add text to label the button"
-                  description="Error message displayed when user tries to save tipline menu option with a blank label"
-                /> : null
-            }
-            fullWidth
-          />
-
-          { noDescription ?
-            null :
-            <Box my={2}>
-              <TextField
-                label={
+                  id="smoochBotMainMenuOption.label"
+                  defaultMessage="Button text - 24 characters limit"
+                  description="Text field label on dialog that opens to add a new option to tipline bot main menu"
+                />
+              }
+              defaultValue={text}
+              onBlur={(e) => { setText(e.target.value); }}
+              componentProps={{ maxLength: 24 }}
+              variant="contained"
+              error={submitted && !text}
+              required
+              helpContent={
+                submitted && !text ?
                   <FormattedMessage
-                    id="smoochBotMainMenuOption.description"
-                    defaultMessage="Description - 72 characters limit"
-                    description="Description field label on dialog that opens to add a new option to tipline bot main menu"
-                  />
-                }
-                defaultValue={description}
-                onBlur={(e) => { setDescription(e.target.value); }}
-                inputProps={{ maxLength: 72 }}
-                variant="outlined"
-                fullWidth
-              />
-            </Box> }
-
-          <Box my={2}>
-            <Typography variant="body1" component="div">
-              <FormattedMessage
-                id="smoochBotMainMenuOption.send"
-                defaultMessage="If the button is clicked, then send:"
-                description="Disclaimer displayed on dialog that opens to add a new option to tipline bot main menu"
-              />
-            </Typography>
-          </Box>
-
-          <StyledFormControl variant="outlined" error={submitted && !value} fullWidth>
-            <InputLabel>
-              <FormattedMessage
-                id="smoochBotMainMenuOption.response"
-                defaultMessage="Response"
-                description="Input label displayed on dialog that opens to add a new option to tipline bot main menu"
-              />
-            </InputLabel>
+                    id="smoochBotMainMenuOption.labelError"
+                    defaultMessage="Please add text to label the button"
+                    description="Error message displayed when user tries to save tipline menu option with a blank label"
+                  /> : null
+              }
+            />
+          </p>
+          <p>
+            <TextField
```

Comment:
is there a reason we aren't using the `src/app/components/layout/inputs/LimitedTextField` component  here?

Reply:
That's a good question. Honestly I was just replacing one for one, so didn't really think it through, but you are correct, with the max length on this, I think it would be better as a limitedtextfield. will address

Reply:
Thanks @dariusk . I have updated the code to use `<LimitedTextArea...` instead.